torch/distributed/rpc/api.py

```diff
@@ -376,13 +400,16 @@ def remote(to, func, args=None, kwargs=None):
     kwargs = kwargs if kwargs else {}
 
     if qualified_name is not None:
-        return _invoke_remote_builtin(
-            _agent, info, qualified_name, rf, *args, **kwargs)
+        return _invoke_remote_builtin(_agent, info, qualified_name, rf, *args, **kwargs)
+    elif isinstance(func, torch.jit.ScriptFunction):
+        return _invoke_remote_torchscript(
+            info, torch._jit_internal._qualified_name(func), *args, **kwargs
```

Comment:
are you planning to add autograd profiler and test in follow up diff?

Reply:
Right, I haven't passed the profiler state `rf` in this PR yet. 
I can do it in a following PR. 
I am feeling the 3 code paths are more compact this way.